pkg/kv/kvserver/batcheval: BenchmarkRefreshRange failed

pkg/kv/kvserver/batcheval.BenchmarkRefreshRange [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20469786?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachprodMicrobenchmarkWeeklyWip/20469786?buildTab=artifacts#./artifacts/microbench/experiment) on master @ [298633de384ce4aa4cf65d7d964422459a7c88a6](https://github.com/cockroachdb/cockroach/commits/298633de384ce4aa4cf65d7d964422459a7c88a6):


```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.setupMVCCPebble
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:181
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.setupData
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:287
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.runRefreshRangeBenchmark
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:117
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.BenchmarkRefreshRange.func1.1
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:103
          | testing.(*B).runN
          | 	GOROOT/src/testing/benchmark.go:193
          | testing.(*B).run1.func1
          | 	GOROOT/src/testing/benchmark.go:215
          | runtime.goexit
          | 	src/runtime/asm_amd64.s:1700
        Wraps: (2) cluster version must be provided if min version file exists
        Error types: (1) *withstack.withStack (2) *errutil.leafError
--- FAIL: BenchmarkRefreshRange/mixed-case/refresh_window=[95.00,99.00]
BenchmarkRefreshRange/mixed-case/refresh_window=[99.00,99.00]
    test_fixtures.go:85: using existing fixture "refresh_range_bench_data_v25.3-upgrading-to-25.4-step-012_random_readonly_1000000_64_256_v2" in "/home/ubuntu/.cache/crdb-test-fixtures/refresh_range_bench_data_v25.3-upgrading-to-25.4-step-012_random_readonly_1000000_64_256_v2"
    cmd_refresh_range_bench_test.go:183: could not initialize fs env at /home/ubuntu/.cache/crdb-test-fixtures/refresh_range_bench_data_v25.3-upgrading-to-25.4-step-012_random_readonly_1000000_64_256_v2: cluster version must be provided if min version file exists
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/storage/fs.ValidateMinVersionFile
          | 	pkg/storage/fs/min_version.go:84
          | github.com/cockroachdb/cockroach/pkg/storage/fs.InitEnv
          | 	pkg/storage/fs/fs.go:206
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.setupMVCCPebble
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:181
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.setupData
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:287
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.runRefreshRangeBenchmark
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:117
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval_test.BenchmarkRefreshRange.func1.1
          | 	pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go:103
          | testing.(*B).runN
          | 	GOROOT/src/testing/benchmark.go:193
          | testing.(*B).run1.func1
          | 	GOROOT/src/testing/benchmark.go:215
          | runtime.goexit
          | 	src/runtime/asm_amd64.s:1700
        Wraps: (2) cluster version must be provided if min version file exists
        Error types: (1) *withstack.withStack (2) *errutil.leafError
--- FAIL: BenchmarkRefreshRange/mixed-case/refresh_window=[99.00,99.00]
--- FAIL: BenchmarkRefreshRange/mixed-case
    cmd_refresh_range_bench_test.go:113: -- test log scope end --
--- FAIL: BenchmarkRefreshRange
FAIL

I250915 18:22:00.119767 1 (gostd) rand.go:268  [-] 1  random seed: 217040840760964266
test logs left over in: /tmp/logBenchmarkRefreshRange2470937804
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*BenchmarkRefreshRange.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-54478

--- a/pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go
+++ b/pkg/kv/kvserver/batcheval/cmd_refresh_range_bench_test.go
@@ -178,11 +178,15 @@ type benchOptions struct {
 type engineMaker func(testing.TB, string, int64, fs.RWMode) storage.Engine
 
 func setupMVCCPebble(b testing.TB, dir string, lBaseMaxBytes int64, rw fs.RWMode) storage.Engine {
-	env, err := fs.InitEnv(context.Background(), vfs.Default, dir, fs.EnvConfig{RW: rw}, nil /* statsCollector */)
+	settings := cluster.MakeTestingClusterSettings()
+	env, err := fs.InitEnv(context.Background(), vfs.Default, dir, fs.EnvConfig{
+		RW:      rw,
+		Version: settings.Version,
+	}, nil /* statsCollector */)
 	if err != nil {
 		b.Fatalf("could not initialize fs env at %s: %+v", dir, err)
 	}
-	eng, err := storage.Open(context.Background(), env, cluster.MakeTestingClusterSettings(), storage.LBaseMaxBytes(lBaseMaxBytes))
+	eng, err := storage.Open(context.Background(), env, settings, storage.LBaseMaxBytes(lBaseMaxBytes))
 	if err != nil {
 		env.Close()
 		b.Fatalf("could not create new pebble instance at %s: %+v", dir, err)

